docs/manual/lore/faction-sirius.md

```diff
@@ -0,0 +1,65 @@
+# Great House Sirius
+
+The House Sirius is one of the major factions in Naev.
+
+## The Facts
+
+* Leader: Sirichana
+* Leading Structure: Sirichana as interpreted by the Serra
```

Comment:
Wouldn't it be the three arch canters?

Reply:
I have no problem, but the doc file for the Sirii and the timeline didn't mention the precise reilgious structure. The file only said, in the section related to the Nasin, that they believe worship can be done without following the Serra.

Reply:
The Nasin stuff needs to get reworked a bit, I'm not sure it's entirely consistent with the Sirius lore, especially how it has been changed recently to be psychic and stuff. I would go by the main document first now.

Reply:
The main document?

Reply:
The non-nasin parts.

Reply:
I've called them the "Tribunal of Arch-Canters"

Reply:
I'll resolve the conversation, taking the "Tribunal" to be an acceptable title. Please open highlighting the new doc to handle that.